integration/show_templates_test.go

```diff
@@ -0,0 +1,3 @@
+package integration
+
+// TODO
```

Comment:
I haven't actually used the template feature yet. Do i need to add a task first, then turn it into a template? Or, can I do this in one go?

Reply:
I think the template command creates a new template from scratch, so you can do it in one go. That means [this commen](https://github.com/naggie/dstask/blob/75615db7f53b16e51a4bd08bce99d9cf7815ef88/commands.go#L570)t is incorrect.